### PR TITLE
Add a function to easily add issue templates in package quillt-related

### PR DESCRIPTION
**Pull request by cderv**
During meeting we talked about simplifying the issue templates in repo. As a reminder, https://github.com/rstudio/blogdown/issues/new/choose

We mention: 

* Do not set tag as untagged issue means that it requires triage
* Reduce options in chooser - no need for documentation for example

I decided it could be a good idea to have them stored here to easily set them and keep a common set of template. 

Suggested changes : 

* 2 templates: Bug and Feature request. They differentiate by the help instruction inside. I think the text should be a comment because
   * It opens Github when you click in source mode anyway, so formatting is not seen
   * This prevent having unrelated text still there in the issue when posted because the user did not remove.
  I also make the text less imperative and more like advice to follow. 
* Keep the links to other site: Issue guide, SO and community - but make that optional in quilt function
* By default does not allow empty issue with no template

This require more documentation for future us, but I open a PR to get your first feedback @apreshill ! 
Thanks!

### CODE REVIEW

**Comment by cderv** [comment]
Tested on **bookdown**: https://github.com/rstudio/bookdown/issues/new/choose

File: inst/templates/issue_template-bug-report.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+name: "Open an issue in this repo"
```
Comment (apreshill) [inline inst/templates/issue_template-bug-report.md:2]:
```suggestion
name: "Bug report"
```

The wording here was unclear to me that it was a bug report- I think most people with GitHub accounts who have made it this far will know this term?

File: inst/templates/issue_template-bug-report.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+about: Give us feedback about your issues or unclear documentation
```
Comment (apreshill) [inline inst/templates/issue_template-bug-report.md:3]:
```suggestion
about: Report an error or unexpected behavior you saw while using this package
```

Again wasn't totally clear from wording about feedback/unclear docs that this was a bug report

File: inst/templates/issue_template-config.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+blank_issues_enabled: {{ blank_issue }}
+{{#links}}
+contact_links:
+{{/links}}
+{{#issue}}
+  - name: Issue guide
+    url: {{{ issue.url }}}
+    about: First time here or need a refresh ? Please consult the issue guide before posting.
```
Comment (apreshill) [inline inst/templates/issue_template-config.yml:8]:
```suggestion
    about: First time here or need a refresh? Please consult the issue guide before posting.
```

File: inst/templates/issue_template-feature-request.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+---
+name: "Feature request"
+about: Suggest an idea  for this package and share experience
```
Comment (apreshill) [inline inst/templates/issue_template-feature-request.md:3]:
```suggestion
about: Suggest an idea for this package
```

**Review by apreshill — CHANGES_REQUESTED** [review]

**Comment by cderv** [comment]
Thanks for the wording review, that helps!

**Comment by cderv** [comment]
If it is good, you can merge.

**Review by apreshill — APPROVED** [review]

**Pull request merged**